oppia/static/html/oppia_preview.html

```diff
@@ -0,0 +1,10 @@
+<div style="opacity: 0.7;">
+  <p>
+    This XBlock does not have an inline Studio preview.
```

Comment:
Is this supposed to show up in Studio? I don't see it:

![screen shot 2015-09-30 at 1 47 20 pm](https://cloud.githubusercontent.com/assets/1985317/10201386/cbc77908-6779-11e5-963d-6426b573ef5f.png)